media/windows/ECC-Ecobee/ECC Pycobee Data.py

```diff
@@ -149,6 +149,23 @@
     Also corrected bug in processing of GMail credentials file and included some missing
     imports for the new mail routine for error-aborts.
             Modified by DK Fowler ... 06-Dec-2020       --- v03.11
+<<<<<<< Updated upstream
+=======
+
+    Modified to include new fields returned from the Ecobee API for the thermostat runtime
+    object:  actual_voc, actual_co2, actual_aq_accuracy, and actual_aq_score.  In order to
```

Comment:
I don't see any python code changes to handle these new fields -- is that in the zip file or something?

Reply:
Yes, in the zip.  Probably wasn't necessary to update the core polling routine, but wanted to track it in the comments...

Reply:
Gotcha!